sdks/python/apache_beam/testing/test_stream.py

```diff
@@ -168,7 +198,11 @@ def add_elements(self, elements):
     element.  The windows of a given WindowedValue are ignored by the
     TestStream.
     """
+    self.output_tags.add(tag)
```

Comment:
Do we need to do this if tag is None?

Reply:
Yep, tag=None is a valid tag for a PCollection. It is entirely reasonable for a user to have multiple PCollections with one of them as tag=None.